edith-step/src/ReceiptInput.js

```diff
@@ -3,27 +3,57 @@ import React from 'react';
 export default class ReceiptInput extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {items: [], itemName: '', itemPrice: 0.0};
+    this.state = {items: [], itemName: '', itemPrice: 0.0, itemQuantity: 1, 
+                  };
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.getDate = this.getDate.bind(this);
+  }
+
+  getDate() {
```

Comment:
javadocs on getDate() and handleSubmit() for readability

Reply:
Done.